app/src/docs/v1.api-spec.yaml

```diff
@@ -155,7 +163,121 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /status:
+    parameters:
+      - $ref: '#/components/parameters/StatusFilter'
+    get:
+      summary: Queryable message transaction status
+      operationId: GetStatusQuery
+      tags:
+        - Status
+      security:
+        - OpenID:
+            - EMAILER
+      parameters:
+        - in: query
+          name: tag
+          description: The desired messages with the specified tag association
+          schema:
+            type: string
+          example: tag
+        - in: query
+          name: txId
+          description: The desired messages associated with the transaction uuid
+          schema:
+            type: string
+            format: uuid
+          example: 00000000-0000-0000-0000-000000000000
+        - in: query
+          name: status
+          description: Yield messages in the specified status
+          schema:
+            type: string
+            enum:
+              - active
```

Comment:
Are we going to expose all the implementation stages or just ones relevant to business?  like what do each of those statuses mean?  we will have to document that.

Reply:
Not sure to be honest about this just yet. All of those stages are currently in the bull queue, but if we want to mask away some of those states, we're going to need to map some of them to "business" terminology.

Reply:
I think it's better to define our business and API, not just surface Bull.  We have no idea if Bull will be viable long term.  We can map Bull to our API.

Reply:
Once there's a separate persistace layer on top of Bull can have mappings between queue state and business state if we want

Reply:
In the short term then, would it be fine to just say we do a quick passthrough, and mark the field as a DRAFT so that we know that it hasn't been finalized yet?